src/frontend/pages/ProjectDetailPage/project-view-container/risk-log/risk-log.tsx

```diff
@@ -0,0 +1,16 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import CheckList from '../../../../components/check-list/check-list';
+
+interface RiskLogProps {
+  risks: { details: string; resolved: boolean }[];
```

Comment:
It might be worth adding this risk type to the types folder in utils as it's probably going to be used in several places once we actually hook up the feature. Thoughts @jamescd18?

Also the feature looks great btw! All my comments are just knitpicky stuff.

Reply:
definitely agree with this, there actually should be one already